GiveNGo/GiveNGo
# Navigation

## Description: 
* Initial setup of Navigation (Routes) to route to different pages within app. 

## Changes I Made:
* Added dependencies for react-native and react-navigation to package.json
* Imports to App.tsx for NavigationContainer and createStackNAvigator
* Created Containers files for each container
* Added Stacks to the NavigationContainer in App component
* Created simple on press navigation to LoginPage to direct to SignupPage

## How to Test: 
* `npm install`
* `npm start`
* You should see a page with Login title and button for Signup that will direct to signup page on press action
